templates/taxi/driver_list.html

```diff
@@ -1,7 +1,8 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
+    <h1> Driver List
+      <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">+</a>
     </h1>
```

Comment:
Ensure that the `<h1>` tag is properly closed. Currently, it seems to be incorrectly structured, which may lead to invalid HTML.